src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/privateApi/LicenceController.kt

```diff
@@ -763,85 +763,6 @@ class LicenceController(
     return licenceService.updateLicenceStatus(licenceId, request)
   }
 
-  @Tag(name = Tags.JOBS)
-  @Deprecated(
-    message = "This endpoint is being deprecated as it is believed to no longer be in use after the activation job was moved to the API",
-  )
-  @PostMapping(value = ["/activate-licences"])
-  @PreAuthorize("hasAnyRole('SYSTEM_USER', 'CVL_ADMIN')")
-  @Operation(
-    summary = "Activate licences in bulk",
-    description = "Set licence statuses to ACTIVE. Accepts a list of licence IDs. Requires ROLE_SYSTEM_USER or ROLE_CVL_ADMIN.",
-    security = [SecurityRequirement(name = "ROLE_SYSTEM_USER"), SecurityRequirement(name = "ROLE_CVL_ADMIN")],
```

Comment:
Deleted as no longer used

Reply:
I take it there aren't any controller tests associated with this route which also need to be deleted?